ledger/metrics/src/main/scala/com/daml/metrics/VarGauge.scala

```diff
@@ -10,5 +10,6 @@ import com.codahale.metrics.Gauge
 case class VarGauge[T](initial: T) extends Gauge[T] {
   private val ref = new AtomicReference[T](initial)
   def updateValue(x: T): Unit = ref.set(x)
+  def updateValue(up: T => T): Unit = ref.updateAndGet(up(_))
```

Comment:
Looks like you need to explicitly return `()`:

```suggestion
  def updateValue(up: T => T): Unit = {
    ref.updateAndGet(up(_))
    ()
  }
```

Might want to consider adding whitespace around the methods too.

Reply:
Sorry for the compile errors ... I haven't installed nix since I've upgraded to Catalina, so I can't currently build the daml repo locally 🙈

Reply:
No worries. Feel free to add the "automerge" label if you're happy. I am. 🙂